www/site/content/docs/chef-workstation/chef-run-users-guide.md

```diff
@@ -0,0 +1,159 @@
++++
+title = "chef-run User's Guide"
```

Comment:
Proposed Title - the title in THe Big Doc was "examples"

Reply:
I think we can drop _User's_